src/mrun

```diff
@@ -1,11 +1,18 @@
-#!/bin/sh
+#!/bin/sh -x
 
 [ $# -lt 2 ] && echo "usage: $0 <name> <command> [params...]" && exit 1
 
 root=`dirname $0`
 run_name=$1
 command=$2
+CEPH_BIN=$root
+CEPH_CONF_PATH=$root/run/$run_name
+
+if [ -e CMakeCache.txt ]; then
+    CEPH_BIN=$PWD/src
```

Comment:
note that this bit is going to change to `$PWD/bin` once @alimaredia's cmake check work (https://github.com/ceph/ceph/pull/7912) gets merged